model/src/main/scala/za/co/absa/atum/model/RunStatus.scala

```diff
@@ -15,10 +15,15 @@
 
 package za.co.absa.atum.model
 
+import com.fasterxml.jackson.core.`type`.TypeReference
+import com.fasterxml.jackson.module.scala.JsonScalaEnumeration
 import za.co.absa.atum.model.RunState.RunState
 
+class RunStateType extends TypeReference[RunState.type]
+
 case class RunStatus
 (
+  @JsonScalaEnumeration(classOf[RunStateType])
```

Comment:
**Just a note:** The whole class feels little utilitarian for Enceladus' purposes. But no point of removing it now..

Reply:
No such class in the next major version 👍